### PR TITLE
Add support for the new HTTP/2 based Apple push notification service

### PR DESCRIPTION
**Pull request by pmeulen**
Add a APNS2 message class using [edamov/pushok](https://github.com/edamov/pushok) that supports sending push notifications using the new Apple HTTP/2 api

Update the TestServer to use APNS2.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
         "ext-curl": "*",
         "ext-json": "*",
         "php": ">=7.2",
-        "psr/log": "^1.1"
+        "psr/log": "^1.1",
+        "edamov/pushok": "^0.14.0",
```
Comment (MKodde) [inline composer.json:14]: Albeit small, this seems the go-to PHP library when it comes to apple push notifications :+1: . I was surprised no other bigger vendors provided this functionality.

File: library/tiqr/Tiqr/Message/APNS2.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            'certificate_path' => $cert_filename,
+            'certificate_secret' => null // Private key secret
+        ];
+
+        $authProvider = AuthProvider\Certificate::create($authProviderOptions);
+
+        // Create the push message
+        $alert=Alert::create();
+        $alert->setBody($this->getText());
+        // TODO: Set title?
```
Comment (MKodde) [inline library/tiqr/Tiqr/Message/APNS2.php]: The `Tiqr_Message_Abstract` does not implement a getter or a field for the title of the push notification message. Is this something you'd like to implement in this PR give you left a todo comment? Or is this a marker for future development. In that case my personal preference is to remove the TODO signs and move them into Pivotal.

/Nitpick

File: library/tiqr/Tiqr/Message/APNS2.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        $payload=Payload::create()->setAlert($alert);
+        $payload->setSound('default');
+        // TODO: Can we set an expiry time for the message?
```
Comment (MKodde) [inline library/tiqr/Tiqr/Message/APNS2.php]: The Expiry time can be set on the Notification I noticed while looking at the library. 

https://github.com/edamov/pushok/blob/master/src/Notification.php#L138

**Review by MKodde — APPROVED** [review]
Looks good from a code review perspective. Some comments and nitpicks can be found below.

File: library/tiqr/Tiqr/Message/APNS2.php (hunk 2 above)
Comment (pmeulen) [reply library/tiqr/Tiqr/Message/APNS2.php]: https://www.pivotaltracker.com/story/show/183081723

File: library/tiqr/Tiqr/Message/APNS2.php (hunk 3 above)
Comment (pmeulen) [reply library/tiqr/Tiqr/Message/APNS2.php]: Thanks, I missed that!

Added expiry.

**Pull request merged**